src/pages/workspace/reimburse/WorkspaceRateAndUnitPage/RatePage.tsx

```diff
@@ -49,21 +47,10 @@ function WorkspaceRatePage(props: WorkspaceRatePageProps) {
         Navigation.navigate(ROUTES.WORKSPACE_RATE_AND_UNIT.getRoute(props.policy?.id ?? ''));
     };
 
-    const validate = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_RATE_AND_UNIT_FORM>): FormInputErrors<typeof ONYXKEYS.FORMS.WORKSPACE_RATE_AND_UNIT_FORM> => {
-        const errors: FormInputErrors<typeof ONYXKEYS.FORMS.WORKSPACE_RATE_AND_UNIT_FORM> = {};
-        const rate = values.rate;
-        const parsedRate = MoneyRequestUtils.replaceAllDigits(rate, toLocaleDigit);
-        const decimalSeparator = toLocaleDigit('.');
-        const outputCurrency = props.policy?.outputCurrency ?? CONST.CURRENCY.USD;
-        // Allow one more decimal place for accuracy
-        const rateValueRegex = RegExp(String.raw`^-?\d{0,8}([${getPermittedDecimalSeparator(decimalSeparator)}]\d{1,${CurrencyUtils.getCurrencyDecimals(outputCurrency) + 1}})?$`, 'i');
-        if (!rateValueRegex.test(parsedRate) || parsedRate === '') {
-            errors.rate = 'workspace.reimburse.invalidRateError';
-        } else if (NumberUtils.parseFloatAnyLocale(parsedRate) <= 0) {
-            errors.rate = 'workspace.reimburse.lowRateError';
-        }
-        return errors;
-    };
+    const validate = useCallback(
+        (values: FormOnyxValues<typeof ONYXKEYS.FORMS.POLICY_CREATE_DISTANCE_RATE_FORM>) => validateRateValue(values, outputCurrency, toLocaleDigit),
+        [outputCurrency, toLocaleDigit],
+    );
```

Comment:
Please test this change at least on one platform to ensure that nothing broken.
I am testing this also.

Reply:
https://github.com/Expensify/App/assets/23487612/05c7e46f-a3e1-4c2d-a5d8-f4766e967c29

I did. Just tested it again. Is something wrong?